docs/table/construct_table.rst

```diff
@@ -471,6 +471,41 @@ To create a copy of an existing table that is empty (has no rows)::
  float64 float64
  ------- -------
 
+Empty array of a known size
+---------------------------
+
+If you do know the size your table will be, but don't know the values in
+advance, you can create a zeroed numpy array and build the astropy table from
+it::
+
+  >>> N = 4
```

Comment:
You can make it `N=3` to reduce docs length slightly without reducing comprehension.